packages/vscode-extension/package.json

```diff
@@ -1415,6 +1415,10 @@
     "@azure/identity": "^3.1.3",
     "@azure/ms-rest-nodeauth": "^3.1.1",
     "@azure/msal-node": "^1.14.6",
+    "@microsoft/dev-tunnels-connections": "^1.0.7342",
```

Comment:
Are these packages stable enough?
Consider fixed version (no `^`) to avoid compatibility issue if there will be breaking change

Reply:
Sure, and I will update the package version after dev tunnel GA.